ScarletEngine/Sources/Gameplay/Public/World.h

```diff
@@ -63,10 +68,7 @@ namespace ScarletEngine
 			ZoneScoped
 			for (const auto& Sys : Systems)
 			{
-				for (const SharedPtr<Entity>& Ent : Entities)
-				{
-					Sys->Run(Ent->ID);
-				}
+				Sys->Update(Entities, GetDeltaTime());
```

Comment:
Avoid calling `GetDeltaTime` every loop by creating a temporary `const double Dt` outside the loop and passing that as the parameter instead.